app/controllers/events_controller.rb

```diff
@@ -0,0 +1,86 @@
+class EventsController < ApplicationController
```

Comment:
Protect that controller. 
Only connected user must be able to have access to its actions
Use `before_action` to do that

Reply:
It's possible to access to the event creation form without being connected even if it's generated an error
```rb

20 21 22 23 24 25 | # GET /events/new   def new     @event = current_user.events.build   end    # GET /events/1/edit
-- | --



```